easybuild/easyconfigs/s/Seurat/Seurat-3.1.2-foss-2019a-R-3.6.0.eb

```diff
@@ -0,0 +1,54 @@
+easyblock = 'Bundle'
+
+name = 'Seurat'
+version = '3.1.2'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://satijalab.org/seurat'
+description = "Seurat is an R package designed for QC, analysis, and exploration of single cell RNA-seq data."
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+dependencies = [
+    ('R', '3.6.0'),
+    ('R-bundle-Bioconductor', '3.9', versionsuffix),
+]
+
+exts_default_options = {
+    'source_urls': [
+        'https://cran.r-project.org/src/contrib/Archive/%(name)s',  # package archive
+        'https://cran.r-project.org/src/contrib/',  # current version of packages
+        'https://cran.freestatistics.org/src/contrib',  # mirror alternative for current packages
+    ],
+    'source_tmpl': '%(name)s_%(version)s.tar.gz'
+}
+
+exts_defaultclass = 'RPackage'
+exts_filter = ("R -q --no-save", "library(%(ext_name)s)")
+
+exts_list = [
+    ('future.apply', '1.3.0', {
+        'checksums': ['6374eca49bb81e05c013509c8e324cf9c5d023f9f8217b29ce7b7e12025ca371'],
+    }),
+    ('leiden', '0.3.1', {
+        'checksums': ['17fa1e49667fdd30ef5166506181c8514ae406f68f0878a026ee111bff11f8a5'],
+    }),
+    ('sctransform', '0.2.0', {
+        'checksums': ['d7f4c7958693823454f1426b23b0e1e9c207ad61a7a228602a1885a1318eb3e4'],
+    }),
+    ('uwot', '0.1.5', {
+        'checksums': ['3253b44c5c7d9ecff25511ad1632a8e924677f9e9a1a8453088de09e90e2e95f'],
+    }),
+    (name, version, {
+        'checksums': ['446a5baf63efdb30ef86af011f94e989597d72787f1d70c39eb32eabd3883854'],
+    }),
+]
+
+modextrapaths = {'R_LIBS': ''}
```

Comment:
@smoors why did you add this? Is it because it's a bundle now instead of a package?

Reply:
@verdurin indeed. R_LIBS is automatically set for RPackage, but not for Bundle.
I guess we could create an RBundle to fix this.